py/client/pydeephaven/_console_service.py

```diff
@@ -40,6 +42,7 @@ def run_script(self, server_script):
             raise DHError("failed to execute a command in the console.") from e
 
     def open_table(self, name):
```

Comment:
typehint and return value doc

... stopping review here, since all of the comments are the same.